pkg/api/types.go

```diff
@@ -46,9 +49,12 @@ func ProtoLvolToLvol(l *spdkrpc.Lvol) *Lvol {
 }
 
 func LvolToProtoLvol(l *Lvol) *spdkrpc.Lvol {
+	if l == nil {
+		return nil
+	}
 	return &spdkrpc.Lvol{
-		Name:         l.Name,
-		Uuid:         l.UUID,
+		Name: l.Name,
+		// Uuid:         l.UUID,
```

Comment:
Can we add `UserCreated` flag? Currently, the flag is always true before finishing  https://github.com/longhorn/longhorn/issues/7578 .

Reply:
Do we need to handle this field in the current PR? I prefer to tackle it later.

Reply:
Sure!

Reply:
@DamiaSan is working on it on SPDK part, so he should be able to help with longhorn integration here.